_projects/bus_routing.md

```diff
@@ -0,0 +1,70 @@
+---
+# Remove the project attributes which don't apply to your project. (ie: if no issues, delete the issues section)
+
+published: false # change to true once ready to post
+
+title: "School Bus Routing Optimization"# project title inside quotes
+excerpt: "Optimize school bus routing plan using general constraint solver" # shows on project list page
+seo_description: "" # goes in project meta description
+
+# project attributes
+requirements: # bullet list of requirements – one requirement per line – follow below format
+  - "Python"
+  - "Data analysis and visualizations skills"
+  - "Java Scala preferred"
+  - "Scala preferred"
+  - "Experience with graph algorithms a plus"
+  - "Knowledge of spatial statistics a plus"
+
+
+tags: # one tag per line – spaces are allowed in tags – use tags other posts use
+  - "python"
+  - "vehicle routing problem"
+  - "OpenTripPlanner"
+  - "Google Optimization Tools"
+
+difficulty: "easy" # easy, medium, hard – pick one
+
+mentors: # github username without the @ – example: designmatty
+  - "simonkassel"
+  - ""
+
+# This file uses Kramdown. See https://kramdown.gettalong.org/syntax.html for syntax
+---
+
+Azavea recently worked with the School District of Philadelphia on a project to investigate the benefits of using an algorithm to create an optimal bus routing plan the school system. If implemented correctly, this type of optimization could lead to dramatic financial and time savings for the district and local students, respectively. Azavea conducted an analysis using [OptaPlanner](https://www.optaplanner.org/) as a general constraint solver and [OpenTripPlanner](https://github.com/opentripplanner/OpenTripPlanner) for vehicle routing. This [effort has been open-sourced](https://github.com/azavea/bus-plan) in the interest of creating a reproducible bus routing tool for school districts accross the country.
+
+The objectives of this project are three-fold:
+1. Augment solver functionality
+2. Analyze results
+3. Develop tool
+
+# MILESTONES
+
+## PREPARATION
+
+- Familiarization with existing BusPlan code
+- Explore the pros and cons of migrating project from OptaPl- anner to [Google Optimization Tools](https://developers.google.com/optimization/)
```

Comment:
@lossyrob I don't know how you feel about this Google idea. i know you had talked about potentially using this in the future. This fellowship may or may not be the right time for it